### PR TITLE
removed unused code

### PR DESCRIPTION
**Pull request by colesnodgrass**
## What
- in commons
    - remove unused code
    - replace old code patterns with newer java functionality
    - ran `./gradlew format`

### CODE REVIEW

**Review by benmoriceau — APPROVED** [review]
Can we check that cloud won't be broken by that change?

**Comment by colesnodgrass** [comment]
@benmoriceau that's what I'm doing right now

**Comment by colesnodgrass** [comment]
I was able to run the cloud locally without any issues.

File: airbyte-commons/src/main/java/io/airbyte/commons/json/JsonSchemas.java [hunk 1]
```diff
@@ -58,7 +58,8 @@ public class JsonSchemas {
    * @param jsonNode - a json object with children that contain types.
    */
   public static void mutateTypeToArrayStandard(final JsonNode jsonNode) {
-    if (jsonNode.get(JSON_SCHEMA_TYPE_KEY) != null && !jsonNode.get(JSON_SCHEMA_TYPE_KEY).isArray()) {
+    if (jsonNode.get(JSON_SCHEMA_TYPE_KEY) != null && !jsonNode.get(JSON_SCHEMA_TYPE_KEY)
```
Comment (cgardens) [inline airbyte-commons/src/main/java/io/airbyte/commons/json/JsonSchemas.java:61]: @colesnodgrass i think your IDE is using a shorter line length than our standard. can you make sure it is set to 150 ([docs](https://docs.airbyte.com/contributing-to-airbyte/code-style))

File: airbyte-commons/src/main/java/io/airbyte/commons/lang/CloseableShutdownHook.java [hunk 2]
```diff
@@ -7,22 +7,21 @@
...
- *
+ * <p>
  * This is a temporary class that is being provided to ensure that resources created by each
  * application are properly closed on shutdown. This logic will no longer be necessary once an
  * application framework is introduced to the project that can provide object lifecycle management.
  */
 public class CloseableShutdownHook {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(CloseableShutdownHook.class);
+  private static final Logger log = LoggerFactory.getLogger(CloseableShutdownHook.class);
```
Comment (cgardens) [inline airbyte-commons/src/main/java/io/airbyte/commons/lang/CloseableShutdownHook.java:24]: why switch from all caps to all down case?

**Review by cgardens — APPROVED** [review]
the deletions look good. can we fix the short line break before merging, please?

File: airbyte-commons/src/main/java/io/airbyte/commons/json/JsonSchemas.java (hunk 1 above)
Comment (colesnodgrass) [reply airbyte-commons/src/main/java/io/airbyte/commons/json/JsonSchemas.java:61]: We actually have two different settings. In the docs we use `150` but in the style-sheet we set [comment lines to 100](https://github.com/airbytehq/airbyte/blob/74792c133677175e0871f49f61b0401cad17eabe/tools/gradle/codestyle/java-google-style.xml#L85). Looks like for some reason the `100` length was also being used for non-comment lines.

We should probably increase the comment-line length to match the `150` value.

File: airbyte-commons/src/main/java/io/airbyte/commons/lang/CloseableShutdownHook.java (hunk 2 above)
Comment (colesnodgrass) [reply airbyte-commons/src/main/java/io/airbyte/commons/lang/CloseableShutdownHook.java:24]: The `@SLF4j` annotation generates a lower case log variable:
```
Example:
  @Slf4j
  public class LogExample {
  }
  
will generate:
  public class LogExample {
      private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LogExample.class);
  }  
```
And I've been stripping out this annotation and replacing it with non-generated `log` code. We presently have a mix of `log` and `LOGGER` and should standardize on one and update everything to match.

**Review by jdpgrailsdev — APPROVED** [review]
:shipit:

File: airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java [hunk 3]
```diff
@@ -1120,14 +1214,19 @@ protected void runSyncAndVerifyStateOutput(final JsonNode config,
                                              final ConfiguredAirbyteCatalog catalog,
                                              final boolean runNormalization)
       throws Exception {
-    final List<AirbyteMessage> destinationOutput = runSync(config, messages, catalog, runNormalization);
-    final AirbyteMessage expectedStateMessage = MoreLists.reversed(messages)
+    final List<AirbyteMessage> destinationOutput = runSync(config, messages, catalog,
+        runNormalization);
+
+    Collections.reverse(messages);
```
Comment (Leibnizhu) [inline airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java:1220]: `Collections.reverse` changes the order of `messages`.  `MoreLists.reversed` generates a new List with reversed messages, and the order of origin `messages` don't change.   

this code change may cause some integration/acceptance test fail

**Pull request merged**